integrations/openai_middleware.py

```
@@ -46,9 +49,73 @@
+    logger.info(
+        f"SuperCompress: {result.original_tokens}→{result.kept_tokens} tok "
+        f"({savings:.1f}% saved) — policy={result.policy_name}"
+    )
+
+    compressed_content = (
+        f"[SuperCompress: {result.original_tokens}→{result.kept_tokens} tok, "
+        f"{savings:.1f}% saved]\n\n"
+        f"{result.compressed_text}\n\n---\n\n{query}"
+    )
+
+    return system_msgs + [{"role": "user", "content": compressed_content}]
```

Ruff (0.16.1)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)